src/rfc3986/parseresult.py

```diff
@@ -83,6 +84,14 @@ class ParseResult(
     urlparse.ParseResult class.
     """
 
+    scheme: Optional[str]
+    userinfo: Optional[str]
```

Comment:
It would be better to use typing.NamedTuple

Reply:
I perfer not because that would require more change on `ParseResultMixin` and `ParseResult`